grafana/overlays/moc/smaug/dashboards/operate-first/pulp-metrics.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  labels:
+    app: grafana
```

Comment:
what about `version` and `component` labels? to be consistent with the other CRDs?

Reply:
well, I copied mine from the `opf-availability.yaml` in the same dir... so, it is consistent with that one :innocent: 

I see that the dashboards related to thoth have a `version` and a `component` (= thoth-station), so I am adding a `version` now... but it's unclear to me what `component` I should use.

There are no guidelines around labels in https://www.operate-first.cloud/apps/content/grafana/add_grafana_dashboard.html, and it seems that the thoth dashboards are the only ones that currently specify a `component`... but pulp is not specific to thoth.

Any suggestions for `component`?

Reply:
no problem @codificat, when I talk about consistency, I think about Thoth usually :) 

I think `component` label is also a usual label we have across all thoth components. Maybe @harshad16 has more insights here :)